frontend/src/app/components/modals/confirm-dialog/confirm-dialog.modal.html

```diff
@@ -1,33 +1,46 @@
-<div class="op-modal--portal confirm-form-submit--modal  ngdialog-theme-openproject">
```

Comment:
Please do not remove theses classes: Both can be used for theming all modals/all confirmation modals, but only if all respective modals have this class.